sdk/python/tests/integration/materialization/test_snowflake.py

```diff
@@ -105,23 +126,62 @@ def test_snowflake_materialization_consistency_external():
         join_keys=["driver_id"],
     )
 
+    schema = [
+        Field(name="driver_id", dtype=Int32),
+        Field(name="value", dtype=feast_dtype),
+    ]
     driver_stats_fv = FeatureView(
         name="driver_hourly_stats",
         entities=[driver],
         ttl=timedelta(weeks=52),
+        schema=schema,
```

Comment:
These arrays only work with specified schemas (we can't infer what the type is from a plain `Array` in Snowflake).